packages/android_intent/android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.androidintent">
+  <uses-sdk android:minSdkVersion="16" />
```

Comment:
Could you please update the CHANGELOG.md of each plugin and increment their versions?